content/sensu-go/5.16/reference/agent.md

```diff
@@ -1305,6 +1306,14 @@ $ sudo systemctl restart sensu-agent
 
 {{< /language-toggle >}}
 
+## Use environment variables with the Sensu agent
+
+After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to your Sensu resources.
+This includes your checks and plugins.
+
+For example, if you configure a `SENSU_TEST_VAR` variable in your sensu-agent file, it will be available to use in your check configurations as `$SENSU_TEST_VAR`.
+
+The environment variables you provide in your sensu-agent file are also available for plugin processes.
```

Comment:
This line seems redundant as on line 1312 it is mentioned that this works for plugins as well. Line 1314 is needed as it shows a more concrete example how to use them.

Reply:
Resolving as it has been removed.